input/vocabulary/codesystem-place-of-service.json

```diff
@@ -0,0 +1,148 @@
+{
+    "resourceType": "CodeSystem",
+    "id": "place-of-service",
+    "url": "https://signalbhn.org/fhir/CodeSystem/place-of-service",
+    "name": "PlaceOfService",
+    "title": "Place Of Service",
+    "status": "draft",
+    "experimental": false,
+    "date": "2023-08-16T16:08:23+05:30",
+    "description": "CMS maintains a list of Place of Service (POS) codes that indicate where a service can be provided.",
+    "caseSensitive": true,
+    "valueSet": "https://signalbhn.org/fhir/ValueSet/place-of-service",
+    "content": "complete",
+    "concept": [
+        {
+            "code":"2",
```

Comment:
Need to retain Leading Zeroes

Reply:
Retained the Leading Zeroes, please review @enjoysparkling